opentofu/modules/state/s3.tf

```
@@ -0,0 +1,58 @@
+resource "aws_s3_bucket" "state_bucket" {
```

Falls es nicht zu viele Änderungen verursacht, könnte man Redundanz in den Ressource-Namen vermeiden, wenn z.B. schon aus dem Ressource-Typ klar wird, dass es ein Bucket ist

```suggestion
resource "aws_s3_bucket" "state" {
```

---

bzw 
```suggestion
resource "aws_s3_bucket" "remote_state" {
```